notification-service/pom.xml

```diff
@@ -185,6 +186,14 @@
             <artifactId>jackson-databind-nullable</artifactId>
             <version>${jackson-databind-nullable-version}</version>
         </dependency>
+        <!-- START Metrics Dependencies -->
```

Comment:
Do you think it makes sense to follow https://github.com/parodos-dev/parodos/pull/208

Reply:
not really, the thing is only used on workflow-service and notification-service. so I do not see a need.